portal/v2/src/Request.tsx

```diff
@@ -114,3 +115,15 @@ export const RequestKubeconfig = async (
     return OnError(err)
   }
 }
+
+export const FetchStatistics = async (cluster: ICluster, statisticsName: string, duration: string, endDate: Date): Promise<AxiosResponse | null> => {
+  duration = convertTimeToHours(duration)
+  try {
+    const result = await axios(
+      "/api/" + cluster.subscription + "/" + cluster.resourceGroup + "/" + cluster.name + "/statistics/" + statisticsName + "?duration=" + duration + "&endtime=" + endDate.toJSON())
```

Comment:
This is a great place for using a javascript template literal to reduce the line size.

Reply:
Ohh yes, thanks for that. I have changed it.